onramp_compassion/mappings/correspondence_mapping.py

```diff
@@ -146,3 +136,22 @@ def _process_odoo_data(self, odoo_data):
             del odoo_data['correspondant_id']
             if sponsorship:
                 odoo_data['sponsorship_id'] = sponsorship.id
+        # Replace dict by a tuple for the ORM update/create
+        if 'page_ids' in odoo_data:

```

Comment:
I think this is not generic enough, because we will have to do the same transformations for all one2many fields we have.
I would rather move this in `base_mapping` and change the test to check any field ending with `_ids`. This will include `page_ids` and work for any other field.